index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" type="text/css" />
+    <title>Review your project</title>
+  </head>
+  <body>
+    <div id="title">
+      Review your project
+      <div id="potato">
+        In this project, the main goal is to review this project by myself and
+        then another TSE will have to re-review my review in order to check that
+        everything is fine or if there are still missing changes that need to be
+        applied before continuing.
+      </div>
+    </div>
+    <div>
+      <img id="tomato" src="assets/review.png" alt="review image" />
+    </div>
```

Comment:
Do not overuse the ```<div>``` tag, for instance, you are missing a global container for all your content which is the ```<main>``` tag that goes just after the ```<body>``` tag. Also, don't forget to use just 1 ```<h1>``` tag per page, you could wrap your title with it in this case to make use of it. This would be a sample structure that would be correct instead of wrapping everything inside divs:

```html
<section>
	<article>
		<figure>
			<img>
			<figcaption></figcaption>
		</figure>
	</article>
</section>
```

Reply:
You really did well to give a sample structure that would help the developer move forward. However, the use of negative statements like `Do not` and `Don't` sounds to be authoritative. You could try this: _`You have overused the <div> tag and it would be better if you use HTML semantics like <main>, <section>, <aside> and so forth instead. Also, it would be better to use just 1 <h1> tag per page`_